motoman_sda10f_support/launch/test_sda10f.launch

```diff
@@ -2,7 +2,7 @@
 <launch>
 	<include file="$(find motoman_sda10f_support)/launch/load_sda10f.launch" />
 	<param name="use_gui" value="true" />
-	<node name="joint_1_state_publisher" pkg="joint_1_state_publisher" type="joint_1_state_publisher" />
+	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
```

Comment:
I have no clue.  TravisCI has been failing due to the catkin tools bug with roslaunch.  It looks like a type that got past the checks.